base/multimedia.jl

```diff
@@ -15,7 +15,7 @@ export AbstractDisplay, display, pushdisplay, popdisplay, displayable, redisplay
 # struct MIME{mime} end
 # macro MIME_str(s)
 import Base: MIME, @MIME_str
-
+import Base64
```

Comment:
Isn't this a bit weird? I thought we moved `Base64` out because nothing in `Base` depend on it?

Reply:
I think stuff in Base depend on it (that's why it is in a bit weird place in sysimg.jl).

Reply:
Why is it needed here? For the doctests? Or is there a method in this file that the test don't cover?